packages/babel-traverse/test/evaluation.js

```diff
@@ -152,6 +152,62 @@ describe("evaluation", function () {
     expect(eval_invalid_call.confident).toBe(false);
   });
 
+  it("should evaluate global call expressions", function () {
+    expect(
+      getPath("isFinite(1);").get("body.0.expression").evaluate().value,
+    ).toBe(true);
+
+    expect(
+      getPath("isFinite(Infinity);").get("body.0.expression").evaluate().value,
+    ).toBe(false);
+
+    expect(
+      getPath("isNaN(NaN);").get("body.0.expression").evaluate().value,
+    ).toBe(true);
+
+    expect(getPath("isNaN(1);").get("body.0.expression").evaluate().value).toBe(
+      false,
+    );
+
+    expect(
+      getPath("parseFloat('1.1');").get("body.0.expression").evaluate().value,
+    ).toBe(1.1);
+
+    expect(
+      getPath("parseFloat('1');").get("body.0.expression").evaluate().value,
+    ).toBe(1);
+
+    expect(
+      getPath("encodeURI('x 1');").get("body.0.expression").evaluate().value,
+    ).toBe("x%201");
+
+    expect(
+      getPath("decodeURI('x%201');").get("body.0.expression").evaluate().value,
+    ).toBe("x 1");
+
+    expect(
+      getPath("encodeURIComponent('?x=1');").get("body.0.expression").evaluate()
+        .value,
+    ).toBe("%3Fx%3D1");
+
+    expect(
+      getPath("decodeURIComponent('%3Fx%3D1');")
+        .get("body.0.expression")
+        .evaluate().value,
+    ).toBe("?x=1");
+
+    // the two tests below produce an undefined result when the node version is less than 16
```

Comment:
```suggestion
```
Since they are features in Babel 8 now.

Reply:
Thank you very much, I forgot to delete that comment